client/my-sites/plugins/plugins-search-results-page/index.jsx

```diff
@@ -93,45 +65,29 @@ const PluginsSearchResultPage = ( {
 				}
 			);
 
-			if ( categoryName ) {
-				title = translate(
-					'Found %(total)s plugin for "%(searchTerm)s" under "%(categoryName)s"',
-					'Found %(total)s plugins for "%(searchTerm)s" under "%(categoryName)s"',
```

Comment:
Why is this being deleted? It was added to get translations in before the functionality was ready. Can we keep this around for the time being?